trade-finance-manager-api/src/v1/controllers/auth/auth-provider-getAuthCodeUrl.api-test.js

```diff
@@ -0,0 +1,125 @@
+const AuthProvider = require('./auth-provider');
+const { REDIRECT_URI } = require('./auth-provider-config');
+
+describe('AuthProvider - getAuthCodeUrl', () => {
```

Comment:
@ttbarnes maybe you want to chip in in testing private helper functions?

Service pattern in NestJS is sligtly different, all functions are in one file and I would prefer it in this way. Testing them directly sometimes is great, but, yes, almost same can be achieved trough public exposure.

Reply:
Interesting topic, I don't have a preference, pros and cons to both IMO.

I would sway towards having the split and improved test coverage; Particularly when AFAIK/remember the TFM API is not exactly publicly exposed, assuming it's locked down very well with the infastructure and security measures. With that in mind I don't see much benefit to changing this; Unless i'm missing something?

If we do want to change this, perhaps it could be a tech improvement in a new ticket/PR, in the interests of progressing SSO.

Reply:
Definately a new ticket if we were ever wanting to change this, I'd imagine if somoene has a stronger preference theyll refactor this as they go.

Thanks for the thoughts -- No benifit in changing this as is, was just wondering what you thought (I'm a fan of never exporting things for unit tests, but that's just me)!